internal/profile/files.go

```diff
@@ -32,15 +33,19 @@ func (cfg simpleFile) configFilesDiffer() (bool, error) {
 	if err != nil {
 		return false, errors.Wrapf(err, "error reading %s", cfg.path)
 	}
-	if string(changes) != cfg.body {
-		return true, nil
+	if string(changes) == cfg.body {
+		return false, nil
 	}
-	return false, nil
+	return true, nil
 }
 
 // writeConfig writes the config item
 func (cfg simpleFile) writeConfig() error {
-	err := os.WriteFile(cfg.path, []byte(cfg.body), 0644)
```

Comment:
Out of curiosity: is that a bug in the current implementation or we haven't created any dirs?

Reply:
We hadn't create any dirs yet, everything else are files directly in the profile directory.